docs/src/introduction/scopes.md

```diff
@@ -0,0 +1,167 @@
+# Scopes
+
+Scopes might be the most important concept in Stack. We have put a ton of thought into how they work and how they should be used. Scopes are the backbone of Stack, and they are what make it just like any other programming language.
```

Comment:
> *'...they are what make it just like any other programming language.'*

Maybe something to the effect of, *'Akin to applicative languages that more people are familiar with.'*.